sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -100,10 +104,11 @@ function start( $module_configs = null ) {
 		 * @since 4.2.0
 		 * @since 7.3.0 Added $range arg.
 		 *
-		 * @param $full_sync_config - array
```

Comment:
Should we fix those `@param`s? I think they should be `@param type $name Description` instead of what we have right now.

Reply:
lets do it.

Reply:
I think you are still missing descriptions here. As a result we have no "parameters" field in the Codex:
https://developer.jetpack.com/hooks/jetpack_full_sync_start/

Reply:
Good point, improved the docs in d8af90be351f13678f2b5368ca13ec37bef07d1c.